src/journal-remote/journal-upload.c

```diff
@@ -52,6 +52,7 @@ static const char *arg_machine = NULL;
 static bool arg_merge = false;
 static int arg_follow = -1;
 static const char *arg_save_state = NULL;
+static long arg_curl_timeout_secs = -1;
```

Comment:
we standardize on time units in µs, and use the `usec_t` type of that (a typedef for `uint64_t`), to make this clear. 

Also, please do not use -1, but USEC_INFINITY to indicate no timeout

Also, do no mention curl in the option, that's an implementation detail and should not leak into the configuration file. i.e.:

```
static usec_t arg_network_timeout_usec = USEC_INFINITY;
```